nicolegooden/potter
# Header

#### What's this PR do?  

+ Adds header to app 
+ Adds header content, such as a logo on the left, and a series of links on the right (home, characters, spells)

#### How should this be manually tested?  

Open the app locally, view the header, go through the sorting hat process, then click on `choose character` to be routed to a different url.  The header should still be visible.

#### Any background context you want to provide?  

+ Decided against any collapsible nav bar styling due to simplicity of the nav links (limited amount)
+ Decided against showing the App's title in the header, as it looked cluttered

#### What are the relevant tickets?  

Closes #14 

#### Screenshots (if appropriate) 

![Screen Shot 2020-11-06 at 7 19 38 PM](https://user-images.githubusercontent.com/62262404/98429627-0dc91600-2065-11eb-93eb-f8048eeb7d6b.png)
